src/lib/processor/process_claim_seat.rs

```diff
@@ -0,0 +1,21 @@
+use ellipsis_client::EllipsisClient;
+use phoenix_sdk::utils::create_claim_seat_ix_if_needed;
```

Comment:
I would prefer this to not be in a module called utils. No huge issue though

Reply:
Sounds good. I can file an issue in linear / add to my notes, along with the SDK refactor for idempotency